app/assets/stylesheets/forever_style_guide/modules/_ambassador_banner.scss

```diff
@@ -42,26 +48,34 @@ a.ambassador_banner-no_ambassador {
     height: 100%;
     background-color: $color-black;
     opacity: 0;
-    @include transition(opacity 0.15s ease-in-out);
+    @include transition(opacity $banner-hover-effect-speed ease-in-out);
     z-index: $zindex-navbar - 1;
     content: "";
     pointer-events: none;
   }
 
   &:hover {
-    text-decoration: none;
+    &::after {
+      opacity: $banner-background-visible;
+    }
+  }
 
+  &.open { 
+    .fa-chevron-down {
+      @include transform(rotate(180deg));
+    }
     &::after {
-      opacity: $ambassador-scrim-opacity;
+      opacity: $banner-background-visible;
     }
   }
 
-  &.open .fa-chevron-down {
-    @include transform(rotate(180deg));
+  .fa-chevron-down {
+    @include transition(all $chevron-rotate-effect-speed linear);
   }
 
-  & .fa-chevron-down {
-    transition: all 0.3s linear;
+  &.ambassador_banner-text-no_ambassador {
+    background-image: none;
+    background-color: $banner-unassigned-bg-color;
```

Comment:
some variables use bg, others using background, probably better if this is consistent:

`$banner-background-visible;`
`$banner-unassigned-bg-color;`